src/main/java/org/generationcp/middleware/ruleengine/naming/expression/SequenceExpression.java

```diff
@@ -47,8 +48,27 @@ public void apply(final List<StringBuilder> values, final AdvancingSource advanc
 						final Matcher matcher = pattern.matcher(upperCaseValue);
 						if (matcher.find()) {
 							final String keyPrefix = upperCaseValue.substring(0, matcher.start());
-							final int nextNumberInSequence =
-								this.germplasmNamingService.getNextNumberAndIncrementSequenceUsingNativeSQL(keyPrefix);
+							int nextNumberInSequence = 0;
+
+							// check if action is preview, do not increment DB value if true
```

Comment:
Change comment to - "// check if action is preview, do not increment sequence value in the database if true"